.github/workflows/build.yml

```diff
@@ -17,4 +28,14 @@ jobs:
       - run: yarn lint
       - run: yarn typescript
       - run: yarn test
-      - run: yarn build
+      - if: github.event.inputs.sa_enable == '' || github.event.inputs.sa_enable == 'true'
```

Comment:
I think we don't need empty condition here, as this property will always be true